flaskServer/restSample.py

```diff
@@ -12,18 +11,16 @@
 class uploadPhoto(Resource):
 	
 	def get(self):
-		return {'status': 'success'}
+		return {'server status': 'server is up and running'}
 		
 
 	def post(self):
 		self.reqparse = reqparse.RequestParser()
```

Comment:
Is there a particular reason why the RequestParser() is set to self.reqparse instead of a local variable?